packages/liveblocks-react-ui/src/components/Comment.tsx

```diff
@@ -481,6 +500,12 @@ export const Comment = forwardRef<HTMLDivElement, CommentProps>(
 
     return (
       <TooltipProvider>
+        {markThreadAsReadWhenVisible && (
+          <MarkThreadAsReadWhenVisibleHandler
+            commentRef={ref}
+            threadId={markThreadAsReadWhenVisible}
```

Comment:
Should we rename the `markThreadAsReadWhenVisible` prop to `autoMarkReadThreadId` maybe?

Reply:
Yep that's probably easier to understand.